primitives/Cargo.toml

```diff
@@ -27,4 +27,7 @@ num-derive = "0.2"
 # Fixtures
 fake = { version = "^1.3", features = ["chrono"] }
 rand = { version = "^0.6" }
-lazy_static = "1.4.0"
\ No newline at end of file
+lazy_static = "1.4.0"
+merkletree = "0.10.0"
+rust-crypto = "^0.2"
```

Comment:
Why did you choose `rust-crypto`? There is stand alone `sha3` crate + it also includes `Keccak`. But if you have a good reason about using `tiny_keccak` I will be ok with that.

But for Sha3 I don't think we need the whole `rust-crypto`